### ISSUE TITLE
Accessibility Review

### ISSUE THREAD

**Issue by jrsalata** [sheepman39/sheepman39.github.io#2]
This issue is a part of an AI-guided research project for a software testing class.

We will be working on performing a review on each of the main sections of the user interface and identify areas in which accessibility might be a concern. This includes color contrast, screen reader compatibility, and overall usability. Those that use various accessibility features should be able to use the website as easily as possible.

Create a report and any code changes about the overall accessibility of the website. However, do not change or create any content itself.
